### DISCUSSION TITLE
How to specify base-paths in specmatic test

### DISCUSSION THREAD

**Discussion by akshat55narang** [specmatic/specmatic#1089] [Ask For Help]
Hi, 

We have a use case in which the base-paths are defined in the servers attribute. Refer to the below screen shot.
![Screenshot 2024-05-03 at 13 31 13](https://github.com/znsio/specmatic/assets/35976537/9476d784-658c-4727-8279-23e6790cb1bb) ![Screenshot 2024-05-03 at 13 43 36](https://github.com/znsio/specmatic/assets/35976537/0c58afea-41c1-4d68-86bf-7ace6cef1fa7)

To run the test on a single yaml, I provide the base-path in --testBaseUrl  parameter itself.
Could you please share how can I run contract tests when the specmatic.json has multiple yaml files each with a different base path ? 

specmatic.json
`{
  "sources": [
    {
      "provider" : "filesystem",
      "test" : [
        "pets.yaml",
        "products.yaml"
      ]
    }
  ]
}
`

Example yaml [products.yaml](https://gist.github.com/akshat55narang/9910f0be822ab95040655b0135d649c2)
[pets.yaml](https://gist.github.com/akshat55narang/1bab95f3e0bf2addc93b22082fee7bc5
)

**Comment by yogeshnikam671** [comment]
Hey @akshat55narang, apologies for the radio silence on this. This is a very good point that you've raised.
We will discuss internally on this and get back to you.
Meanwhile, if you have any proposals on how this can be addressed, please feel free to share.
